### ISSUE TITLE
Issue: When closing the box returns to the top of the post 

### ISSUE THREAD

**Issue by luisspuerto** [archetyped/simple-lightbox#626]
## Description of Problem

Hi! 

I really don't know if this is a issue or I can modify it somehow, but when you closes the box it returns to the top of the post, which is really annoying. 

Can this be changed?

Edit: Grammar

## Details

* URL of Page exhibiting problem: http://luisspuerto.net/blog/ (any post with images) 
* Browser(s) exhibiting problem (Firefox, Chrome, etc.): Safari and Chrome as far as I know. Haven't tried others 
* Version of SLB Installed: 2.6.0
* Other plugins installed: 
   - Akismet Anti-Spam
   - Child Theme Configurator
   - Contact Form 7
   - Crayon Syntax Highlighter
   - Easy Twitter Feed Widget
   - Embed Any Document
   - Google Analytics for WordPress by MonsterInsights
   - Image Widget
   - Jetpack by WordPress.com
   - JP Widget Visibility
   - Open external links in a new window
   - TinyMCE Advanced
   - WP Smush
* Theme: Harmonic 

(Right now I don't have active simple lightbox due to this behaviour, but you can see it [here](http://recordit.co/9xH9DZctbl))

**Comment by luisspuerto** [comment]
No one?

**Comment by archetyped** [comment]
Hi, please fill in the details of your issue as per [the submission guidelines](https://github.com/archetyped/simple-lightbox/wiki/Reporting-Issues), and I would be glad to take a closer look.

Thanks.

**Comment by archetyped** [comment]
Hi, please fill in the details of your issue as per [the submission guidelines](../wiki/Reporting-Issues), and I would be glad to take a closer look.

Thanks.

**Comment by luisspuerto** [comment]
@archetyped I've updated my first comment with that info and a little video of the behaviour. 

Thanks!

**Comment by archetyped** [comment]
Thank you for the additional details.  This type of behavior is sometimes caused by a conflict with a different plugin or the theme.  Here are instructions on [identifying plugin/theme conflicts](https://github.com/archetyped/simple-lightbox/wiki/Troubleshooting-Plugin-Conflicts "identifying plugin/theme conflicts").

Let me know what you find (ideally with a link to the conflicting plugin/theme so that I can test it), and I would be glad to take another look.

Thanks!

**Comment by luisspuerto** [comment]
~~I've deactivate all the plugins and worked perfectly, but I was activating them little by little and I wasn't able to pinpoint which one is the culprit.~~

I've deactivated again all the plugins and still the same behaviour... perhaps it's the theme. 
As I told you before the theme is [Harmonic](https://wordpress.org/themes/harmonic) https://wordpress.org/themes/harmonic/

It's really funny because I deactivate all the plugins in the beginning and your plugin seemed to work well. Then I activated little by little and I couldn't pinpoint which one was the culprit. And now even if deactivate all doesn't work well. So I guess it's the theme. 

However... I have to say that when I installed it worked well and after a while it started to show this behaviour. Or at least seemed to. Now I'm not sure of anything. 

Only thing I can be sure is ... now, having activated only your plugin and nothing more it doesn't work properly.
